src/pages/registration/resource_type_tab/sub_type_forms/artistic_types/architecture/ArtisticArchitectureForm.tsx

```diff
@@ -111,12 +112,12 @@ export const ArtisticArchitectureForm = () => {
           {({ push, replace, remove, move, name }: FieldArrayRenderProps) => (
             <>
               {architectureOutput.length > 0 && (
-                <Table sx={{ '& th,td': { borderBottom: 1 } }}>
+                <Table sx={alternatingTableRowColor}>
                   <TableHead>
-                    <TableRow>
+                    <TableRow sx={{ '& th': { borderBottom: 1 } }}>
```

Comment:
ta vekk disse, ev. flytt til theme?
```suggestion
                    <TableRow>
```